pom.xml

```diff
@@ -565,6 +591,31 @@
         <artifactId>scala-guice_${scala.version.prefix}</artifactId>
         <version>4.0.1</version>
       </dependency>
+      <dependency>
```

Comment:
I think you can actually move these dependencies into the hbase profile as well.